maro/rl/policy/pg.py

```diff
@@ -155,6 +156,36 @@ def learn(self, batch: dict):
     def improve(self):
         self.learn(self._get_batch())
 
+    def distributed_learn(self, batch: dict, worker_id_list: list):
```

Comment:
The meaning of ``distributed_learn`` can be fuzzy. The entire multi-rollout / multi-policy-host architecture can be regarded as learning in distributed fashion. How about we change it to ``learn_with_data_parallel`` to match the ``data_parallel`` interface?